flybywiresim/a32nx
# [LIGHTS] General Improvements | Road Map

## Summary of Changes

#979 #1386 

**_Marked As Fixed_**
- [x] Lights bleed into the cockpit - Fixed a long time ago
- [x] Strobe Lights bleed in the cockpit - Fixed a long time ago.

**_New Features / Improvements_**
- [ ] Make DOME Lights realistic

## Screenshots (if necessary)



### _Coming Soon..._



## References

 ### _Coming Soon..._




## Additional context

_None_


Discord Username: `Snapmatics#0001`


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The `build.py` will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
